tools/onnx_subgraph/docker/Dockerfile.u2204

```diff
@@ -0,0 +1,57 @@
+# Copyright (c) 2025 Samsung Electronics Co., Ltd. All Rights Reserved
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM ubuntu:22.04
+
+ENV DEBIAN_FRONTEND noninteractive
+
+# Install 'add-apt-repository'
+RUN apt-get update \
+ && apt-get -qqy install apt-utils software-properties-common
+
+# Build tool
+RUN apt-get update \
+ && apt-get -qqy install build-essential cmake git fakeroot autoconf automake libtool unzip wget
+
+# additonal tools
+RUN apt-get update \
+ && apt-get -qqy install libprotobuf-dev protobuf-compiler libjsoncpp-dev
+
+# Python
+RUN apt-get update \
+ && apt-get -qqy install python3 python3-pip python3-venv python3-dev python3-all dh-python
+RUN python3 -m pip install --upgrade pip setuptools
+
+ARG VER_ONNX=1.16.0
+ARG VER_ONNXRUNTIME=1.18.1
+ARG VER_ONNXSIM=0.4.36
+ARG VER_TORCH=2.3.1
```

Comment:
required packages and versions are from draft https://github.com/Samsung/ONE/pull/14613/files#diff-8ca7e1f3086f2241d41573341bf57b8e7c2145ef588db9547dc7a71c20122ea7